apps/pie-storybook/stories/pie-cookie-banner.stories.ts

```diff
@@ -5,6 +5,7 @@ import { PieLink } from '@justeattakeaway/pie-link';
 import { PieModal } from '@justeattakeaway/pie-modal';
 import { PieIconButton } from '@justeattakeaway/pie-icon-button';
 import { PieToggleSwitch } from '@justeattakeaway/pie-toggle-switch';
+import { action } from '@storybook/addon-actions';
```

Comment:
QQ – in the storybook docs, it says that `addon-actions` is installed by default as part of the `addon-essentials` package. If that's the case, can e get away with not installing it separately as well?